### PR TITLE
Add hercules' visualization companion labours (python)

### PR DESCRIPTION
**Pull request by dbast**
hercules: Data Science applied to git repos (includes also similar statistics as generate by git-of-theseus).

The go binary is binary repack, which is semi-nice, but compiling it does not provide much more value and requires some significant effort.

All the `core` visualization targets of labours are working (see the tests). There are more targets, which conditionally import more python dependencies, which are available as wheels but not easy to package (seriate -> Google or-tools). Those targets can be enabled by e.g. doing `pip install seriate`).

(Both packages combined work more reliable here than the upstream provided docker image.)

### CODE REVIEW

File: recipes/hercules/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  script:
+    - gunzip hercules.gz               # [unix]
+    - mkdir -p "$PREFIX/bin"           # [unix]
+    - install hercules "$PREFIX/bin/"  # [unix]
+    - mv hercules.exe "%LIBRARY_BIN%\hercules.exe"  # [win]
+
+requirements:
+  build:
+    # Dummy requirement to ensure conda-forge builds this correctly
+    - make
```
Comment (marcelotrevisani) [inline recipes/hercules/meta.yaml]: if you do not need anything just let it empty as
```
build: []
```

File: recipes/labours/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    - labours -i test.pb -o test-def -m devs-efforts
+    - labours -i test.pb -o test-cma -m churn-matrix
+    - labours -i test.pb -o test-ovn -m old-vs-new
+    - labours -i test.pb -o test-own -m ownership
+
+about:
+  home: "https://github.com/src-d/hercules"
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.md
```
Comment (marcelotrevisani) [inline recipes/labours/meta.yaml]: Could you please raise an issue on this project to also package the license?

File: recipes/hercules/meta.yaml (hunk 1 above)
Comment (dbast) [reply recipes/hercules/meta.yaml]: @marcelotrevisani Seems not to work. See last commit... There already is an issue https://github.com/conda-forge/staged-recipes/issues/8377

Comment (dbast) [reply recipes/hercules/meta.yaml]: Ok. Added a revert commit.

File: recipes/labours/meta.yaml (hunk 2 above)
Comment (dbast) [reply recipes/labours/meta.yaml]: Opened https://github.com/src-d/hercules/pull/288

Comment (dbast) [reply recipes/labours/meta.yaml]: removed the recipe

**Comment by dbast** [comment]
Thanks!

**Pull request merged**